Add FAQ for Other properties (#4839)
segmentio/segment-docs: src/connections/destinations/catalog/actions-hubspot-cloud/index.md

> note ""
> A HubSpot Enterprise Marketing Hub account is required to send Custom Behavioral Events.

### Why can't I set an entire object for the Other properties field?

> This destination does not allow selecting an entire object for the Other properties field. HubSpot rejects API calls if a property name does not match with HubSpot's internal name. To prevent the possibility of rejection when working with a large object of key/value pairs, explicit mapping of each key/value pair is required. This helps to ensure that every key matches the pre-created property names in HubSpot.

Reviewer: ```suggestion
This destination doesn't allow selecting an entire object for the Other properties field. HubSpot rejects API calls if a property name doesn't match with HubSpot's internal name. When working with a large object of key/value pairs, map each key/value pair to prevent rejection. This ensures that every key matches the pre-created property names in HubSpot.
```
